webui/react/src/utils/number.test.ts

```diff
@@ -65,4 +65,14 @@ describe('number utility', () => {
       expect(roundToPrecision(0.523843984)).toBe(0.523844);
     });
   });
+
+  describe('findFactorOfNumber', () => {
+    it('should find factors of number', () => {
+      expect(findFactorOfNumber(-12)).toStrictEqual([ 1, 2, 3, 4, 6, 12 ]);
+      expect(findFactorOfNumber(-1)).toStrictEqual([ 1 ]);
+      expect(findFactorOfNumber(0)).toStrictEqual([ ]);
+      expect(findFactorOfNumber(1)).toStrictEqual([ 1 ]);
+      expect(findFactorOfNumber(12)).toStrictEqual([ 1, 2, 3, 4, 6, 12 ]);
```

Comment:
can we had one more test case of:
```
1092 = 2 × 2 × 3 × 7 × 13
```

Reply:
1092 has more factors than those, it's divisor are: 1,2,3,4,6,7,12,13,14,21,26,28,39,42,52,78,84,91,156,182,273,364,546,1092

Added more tests for a larger number with less divisors (to don't have a test resulting too long to respect esling max line length) and a test with a large prime number.